COMPATIBILITY.md

```diff
@@ -47,6 +47,7 @@ Newly filed issues should bear the label `pipeline` for ease of tracking.
 - [X] `HipChatNotifier` (`hipchat`): supported as of 1.0.0
 - [ ] `IronMQNotifier` (`ironmq-notifier`): [JENKINS-35505](https://issues.jenkins-ci.org/browse/JENKINS-35505)
 - [X] `LogParserPublisher` (`log-parser`): supported as of 2.0
+- [X] `RocketChatNotifier` (`rocketchatnotifier`): supported as of 2.0
```

Comment:
What is this 2.0? The wiki says the latest is 0.5.0. And the step exists from at least 0.2.1.

And how is this `RocketChatNotifier`? This is a custom step, which has its own section.